data/events/scripts/player.lua

```diff
@@ -262,6 +253,16 @@ local function useStamina(player)
 	player:setStamina(staminaMinutes)
 end
 
+-- low level experience bonus
+-- bonus is a floating percentage, use 1 for 100% bonus
+local expBonus = {
+	minlevel = 2,
+	maxlevel = 50,
+	bonus = 1
+}
+
+local expBonusDelta = expBonus.maxlevel - expBonus.minlevel -- don't touch
+
```

Comment:
This should be kept at the top, otherwise it will not be found by the regular. `expBonusDelta` maybe can be removed and use `expBonus.maxlevel - expBonus.minlevel` directly

Reply:
> This should be kept at the top, otherwise it will not be found by the regular. `expBonusDelta` maybe can be removed and use `expBonus.maxlevel - expBonus.minlevel` directly

I made the suggested changes, I believe this way everything is fine to be merged!